### DISCUSSION TITLE
Utility of WGCNA with minimal samples?

### DISCUSSION THREAD

**Discussion by aspencoyle** [RobertsLab/resources#1156] [Q&A] [answered]
Hi all!

I'm working my way through the tutorials for running WGCNA, and noticed that within the [FAQ](https://horvath.genetics.ucla.edu/html/CoexpressionNetwork/Rpackages/WGCNA/faq.html), they state "We do not recommend attempting WGCNA on a data set consisting of fewer than 15 samples"

I'm not even close to this - each of my pairwise comparisons has 6 total samples (3 per treatment group), which is...well under the recommended total!

Should I increase sample size by pooling similar treatments together (for instance, pooling Ambient Day 0 and Ambient Day 2)? 

Should I keep my groups as is and run WGCNA despite its limitations with a limited sample size?

 Or should I just look for another tool for this particular job?

**Comment by sr320** [comment, marked as answer]
Run with what you have.

On Thu, Mar 18, 2021 at 6:05 PM afcoyle ***@***.***> wrote:

> Hi all!
>
> I'm working my way through the tutorials for running WGCNA, and noticed
> that within the FAQ
> <https://horvath.genetics.ucla.edu/html/CoexpressionNetwork/Rpackages/WGCNA/faq.html>,
> they state "We do not recommend attempting WGCNA on a data set consisting
> of fewer than 15 samples"
>
> I'm not even close to this - each of my pairwise comparisons has 6 total
> samples (3 per treatment group), which is...well under the recommended
> total!
>
> Should I increase sample size by pooling similar treatments together (for
> instance, pooling Ambient Day 0 and Ambient Day 2)?
>
> Should I keep my groups as is and run WGCNA despite its limitations with a
> limited sample size?
>
> Or should I just look for another tool for this particular job?
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/RobertsLab/resources/discussions/1156>, or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABB4PN3CTZDBCEMVY6WJFOTTEKPI5ANCNFSM4ZNZUI4A>
> .
>
-- 

Steven B. Roberts, Associate Professor

Associate Director - Graduate Program Coordinator

School of Aquatic and Fishery Sciences

University of Washington

Fisheries Teaching and Research (FTR) Building - Office 232

1140 NE Boat Street - Seattle, WA 98105

robertslab.info - ***@***.*** - @sr320

vm:206.866.5141 - cell:360.362.3626

**Comment by aspencoyle** [comment]
Thanks Steven, will do!

For anyone in the future reading this, I found [this guide to WGCNA](https://edu.isb-sib.ch/pluginfile.php/158/course/section/65/_01_SIB2016_wgcna.pdf) really helpful with some of the basic principles - wish I'd checked it out before diving into the tutorials

**Comment by sr320** [comment]
Big picture is we want to see expression clusters (based on temporal
patterns) in individual crabs over time

On Thu, Mar 18, 2021 at 6:37 PM afcoyle ***@***.***> wrote:

> Thanks Steven, will do!
>
> For anyone in the future reading this, I found this guide to WGCNA
> <https://edu.isb-sib.ch/pluginfile.php/158/course/section/65/_01_SIB2016_wgcna.pdf>
> really helpful with some of the basic principles - wish I'd checked it out
> before diving into the tutorials
>
> —
> You are receiving this because you were mentioned.
>
>
> Reply to this email directly, view it on GitHub
> <https://github.com/RobertsLab/resources/discussions/1156#discussioncomment-501066>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ABB4PN2A7IFWB5CMALID5KDTEKTE7ANCNFSM4ZNZUI4A>
> .
>
-- 

Steven B. Roberts, Associate Professor

Associate Director - Graduate Program Coordinator

School of Aquatic and Fishery Sciences

University of Washington

Fisheries Teaching and Research (FTR) Building - Office 232

1140 NE Boat Street - Seattle, WA 98105

robertslab.info - ***@***.*** - @sr320

vm:206.866.5141 - cell:360.362.3626
